src/main/resources/db/migration/V1_83__expense_report_details.sql

```diff
@@ -0,0 +1,190 @@
+--Depends on migration 1_78
+ALTER table juror_mod.financial_audit_details
+    add CONSTRAINT financial_audit_details_id_loc_code_unique UNIQUE (id, loc_code)
+;
```

Comment:
should we drop the existing PK constraint and replace with this one?

Reply:
Removed this constraint in favour for the the PK constrain created in migration 78